lua/sam/modules/sh_pointshop2.lua

```diff
@@ -0,0 +1,102 @@
+sam.permissions.add("pointshop2 manageitems", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 createitems", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 manageusers", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 managemodules", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 exportimport", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 manageservers", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 reset", "Pointshop 2", "superadmin")
+sam.permissions.add("pointshop2 usepac", "Pointshop 2", "superadmin")
+
+sam.command.set_category("Pointshop 2") -- any new command will be in that category unless the command uses :SetCategory function
+ 
+sam.command.new("ps2_addpoints_steamid")
```

Comment:
One usecase for this command is to give points in donation systems that let you run concommands, does SAM let you do this right now? Thinking mostly of this part 

```lua
sam.player.send_message(calling_ply, "{A} gave {V} {S} to {T}", {
                A = calling_ply, V = amount, S = currencyType, T = steamId
            })
```

could you check if running ```pointshop2 addpoints_steamid``` works when run from the server console?

Reply:
Yes, I kept in mind that donation system is main purpose of this command. When issuing from console you need to prefix command with "sam": ![console screen](https://i.imgur.com/IMjX87x.png)
Initially I wanted to hide this command from menu as it's meant to be used mainly for donation system, but I thought it would be less intuitive for user. Second command is meant to mass giveaway points, e.g. for participating in a event.

Reply:
Cool, great job!